static/usage/v7/action-sheet/theming/styling/vue.md

```diff
@@ -29,47 +29,46 @@
 </style>
 
 <template>
-  <ion-button @click="presentActionSheet">Open</ion-button>
+  <ion-button id="open-action-sheet">Open</ion-button>
+  <ion-action-sheet
+    trigger="open-action-sheet"
+    css-class="my-custom-class"
```

Comment:
Since we are using the component directly, we can just specify `class`. This applies to other framework usages (though react should use `className`)
```suggestion
   class="my-custom-class"
```